platforms/software/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/ivy/IvyDynamicRevisionRemoteResolveIntegrationTest.groovy

```diff
@@ -121,7 +121,11 @@ configurations.all {
 }
 
 dependencies {
-    compile group: /${name}/, name: /${name}/, version: "latest.integration"
+    if (${name.contains(':') || name.contains('@')}) {
+        compile(project.dependencyFactory.create(/${name}/, /${name}/, "latest.integration"))
+    } else {
+        compile(/${name}:${name}:latest.integration/)
+    }
```

Comment:
?! what is this monster?

I think the `/$name/` is doing some kind of poor man quoting.   Can we just do the call to create(...) all the time?